pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -31,7 +31,7 @@ const (
 
 	// CustomResourceDefinitionSchemaVersion is semver-conformant version of CRD schema
 	// Used to determine if CRD needs to be updated in cluster
-	CustomResourceDefinitionSchemaVersion = "1.21.1"
+	CustomResourceDefinitionSchemaVersion = "1.21.2"
```

Comment:
This versioning is only used for Cilium [C]CNP. Lines 261-263 are missing this annotation so the "upgrade" of this CRD won't work.

Reply:
Ah good catch. How should we workaround this then?

Reply:
We just need to add the annotation in the Cilium Node. AFAIK, the update logic will update a CRD if the old one does not contain the annotation but the new one does.

Reply:
Presumably we'll need another fix on master to propagate the schema version for 1.9 or later.

Reply:
`master` already attaches the schema versions to all CRDs: https://github.com/cilium/cilium/blob/a7451f82d4c6ed96a9e94d608ec71f4255d55c65/pkg/k8s/apis/cilium.io/v2/client/register.go#L270